examples/embeddable_examples/public/app/register_embeddable.tsx

```diff
@@ -36,17 +39,78 @@ export const RegisterEmbeddable = () => {
           and then selecting <em>Embeddable examples</em> group.
         </p>
+      </EuiText>
+
+      <EuiSpacer size="l" />
+
+      <EuiText>
+        <h2>Show embeddables in the Add panel menu</h2>
         <p>
           Add your own embeddables to <em>Add panel</em> menu by attaching an action to the{' '}
           <strong>ADD_PANEL_TRIGGER</strong> trigger. Notice usage of <strong>grouping</strong> to
           nest related panel types and avoid bloating <em>Add panel</em> menu. Please reach out to
           @elastic/kibana-presentation team to coordinate menu updates.
         </p>
       </EuiText>
-
+      <EuiSpacer size="s" />
       <EuiCodeBlock language="jsx" fontSize="m" paddingSize="m">
         {registerAttachActionSource}
       </EuiCodeBlock>
+
+      <EuiSpacer size="l" />
+
+      <EuiText>
+        <h2>Specify panel placement settings (optional)</h2>
+        <p>
+          Embeddable factories can specify placement settings for their embeddables. This is done by
+          registering placement settings with <strong>registerEmbeddablePlacementStrategy</strong>.
+          The placement settings is a function that takes the serialized state of the embeddable and
+          returns an object with width, height, and strategy properties. The width and height
+          properties are numbers that specify the relative dimensions of the embeddable panel. For
+          example, a DashboardContainer uses <em>grid units</em> where the maximum width is{' '}
+          <EuiCode>48</EuiCode> and the max height is virtually unlimited. The strategy property is
+          a string that specifies the placement strategy to use. Strategies are specific to the
+          consumer. For example, DashboardContainer currently supports the{' '}
+          <EuiCode>placeAtTop</EuiCode> and
+          <EuiCode>findTopLeftMostOpenSpace</EuiCode> strategies. The example below shows how the
+          Field List embeddable developer example specificies its placement settings on a Dashboard.
+        </p>
+      </EuiText>
+      <EuiSpacer size="s" />
+
+      <EuiCodeBlock language="jsx" fontSize="m" paddingSize="m">
+        {registerFieldListEmbeddableSource}
+      </EuiCodeBlock>
+
+      <EuiSpacer size="l" />
+
+      <EuiText>
+        <h2>Saved object embeddables</h2>
+        <p>
+          Embeddable factories, such as Lens, Maps, Links, that can reference saved objects should
+          register their saved object types using{' '}
+          <strong>registerReactEmbeddableSavedObject</strong>. The <em>Add from library</em> flyout
+          on Dashboards uses this registry to list saved objects. The example below could be added
+          to the public start contract for a plugin.
+        </p>
+      </EuiText>
+      <EuiSpacer size="s" />
+
+      <EuiCodeBlock language="jsx" fontSize="m" paddingSize="m">
+        {`
+registerReactEmbeddableSavedObject({
```

Comment:
As a follow-up, could we use raw-loader for this? It could be a dangling file in examples that isn't actually used otherwise - just thinking it would be nice to get type errors on this to keep it up to date.

Reply:
We don't have any saved object embeddable examples, so we can't demo in the flyout. But yeah, I could add dangling file without any other usages. Fixed in https://github.com/elastic/kibana/pull/182120/commits/8329b646df86517a1c783b3da9ee9e29fb5ca799.